discord/ext/commands/view.py

```diff
@@ -114,15 +114,34 @@ def quoted_word(view):
     if current is None:
         return None
 
-    is_quoted = current == '"'
+    # map from opening quotes to closing quotes
+    quotes = {
+        '"': '"',
+        '«': '»',
+        '‹': '›',
+        '“': '”',
+        '‘': '’',
+        '❛': '❜',
+        '❝': '❞',
+        '❮': '❯',
+        '〝': '〞',
+        '＂': '＂',
```

Comment:
This list is missing a lot of other quotes.

Despite the previous comment by @mylbp2ps3 I do not want to support "multiple ending quotes" since at that point the overhead added is too much for little benefit. I want to optimise for the common case not for the exotic one.